src/sections/TechRevolutionSection/withQuery.js

```diff
@@ -7,7 +7,7 @@ const query = graphql`
       relativePath: { eq: "images/team-photo-conversation.jpg" }
     ) {
       image: childImageSharp {
-        fluid(maxHeight: 664) {
+        fluid {
```

Comment:
Why not specify maxWidth here?

Reply:
Default is 800, and we should only work on widths otherwise the images will be croped

Reply:
But this image expands until `1384px`, 800 is not enough .